.github/workflows/update-certifi-cacert.pem.yml

```diff
@@ -0,0 +1,45 @@
+name: Maybe generate certifi cacert.pem update PR
+on:
+  schedule:
+    # 1:00 AM US-Pacific on Wednesdays
+    - cron: '0 8 * * 3'
+  workflow_dispatch:
+jobs:
+  org-check:
+    name: Check GitHub Organization
+    if: ${{ github.repository_owner == 'pex-tool' }}
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Noop
+        run: "true"
+  update-cacert.pem:
```

Comment:
OK, I had to update this to `update-cacert-pem`, but otherwise seems to work - noop'ed since no new certifi releases yet - when I manually ran.